native_client/args.h

```diff
@@ -22,10 +22,12 @@ bool show_times = false;
 
 bool has_versions = false;
 
+bool extended_metadata = false;
```

Comment:
All changes to `args.h` and `client.cpp` should be in a different PR, since we are focusing on API changes ? Or at least a different commit.